static/img/diagrams/deployment.iuml

```diff
@@ -71,9 +71,28 @@ Deployment_Node(aws, "AWS", "Region: eu-west-2") {
       }
     }
 
+    Deployment_Node(mosaic_production_social_care_referrals, "Social Care Referrals (MASH) Data Processing") {
+      Deployment_Node(aws_mosaic_production_S3_proxy_api_gateway, "S3 API Gateway Proxy") {
+        Container(social_care_s3_api_gateway, "Social Care Referrals S3 API", "API Gateway", "Provides API proxy that accepts HTTP requests to add objects to S3")
+        Container(social_care_referrals_aws_s3, "Social Care Referrals S3 Bucket", "Mash form data, JSON", "Stores form data submitted via the MASH referral form")
+      }
+
+      Deployment_Node(aws_mosaic_production_referrals_lambda, "Lambda") {
+        Container(social_care_referrals_lambda, "Social Care Referral Form Data Ingestion", "Typescript", "Provide functionality to take the data sent from Google and process it")
```

Comment:
Maybe "Ingests referrals from S3 and..." something, something passes it or gets the Social Care Case Viewer API to store it. Not sure what the right words are but just a bit more detail than "functionality" and "process".

Might also be worth update the description for the relevant database to say it stores referrals as well.

Reply:
Done in c0a79a6